.github/README.md

```diff
@@ -0,0 +1,166 @@
+## What are GitHub Actions?
+
+GitHub Actions are used for many different purposes.  We use them to run tests in CI, validate PRs are in an expected state, and automate processes.
+
+- [Overview of GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)
+- [What's a workflow?](https://docs.github.com/en/actions/using-workflows/about-workflows)
+- [GitHub Actions guides](https://docs.github.com/en/actions/guides)
+
+## dbt Labs Action Repo
+[dbt-labs/actions](https://github.com/dbt-labs/actions/) is a central repository of actions and workflows dbt uses across repositories.
+
+## General Standards
+
+### Permissions
+- Default to least permissions.
```

Comment:
It might be nice to have a code example of what setting permissions look like. What if I don't set any permissions at all?